docs/sources/flow/reference/config-blocks/tracing.md

```diff
@@ -0,0 +1,56 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/config-blocks/tracing
+title: tracing
+weight: 100
+---
+
+# `tracing` block
```

Comment:
```suggestion
# tracing block
```

Reply:
The rendering of the code tag in the title looks strange. If you make this change, could you do so for the logging topic, too?